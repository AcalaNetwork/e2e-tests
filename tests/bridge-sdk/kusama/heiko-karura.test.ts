import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'heiko',
  to: 'karura',
  token: 'HKO',
})
