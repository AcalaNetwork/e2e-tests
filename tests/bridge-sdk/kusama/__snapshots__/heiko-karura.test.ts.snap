// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'heiko' to 'karura' using bridgeSDK cross-chain 'HKO' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": "(rounded 998)",
  "toChain": "(rounded 1.99)",
}
`;

exports[`'heiko' to 'karura' using bridgeSDK cross-chain 'HKO' > fee 1`] = `"(rounded 0.006)"`;

exports[`'heiko' to 'karura' using bridgeSDK cross-chain 'HKO' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 1000,
  "toChain": 0,
}
`;
